persistent/_timestamp.c

```diff
@@ -152,7 +152,12 @@ TimeStamp_richcompare(TimeStamp *self, TimeStamp *other, int op)
     return result;
 }
 
+
+#ifdef PY3K
+static Py_hash_t
+#else
 static long
+#endif
```

Comment:
I thought about that, but (1) it's only used once and (2) more irkingly I couldn't decide on what a good name would be.